tests/acceptance/test_update_control.py

```diff
@@ -0,0 +1,431 @@
+                "maps": [
+                    {
+                        "id": MUID,
+                        "states": {
+                            "ArtifactInstall_Enter": {
+                                "action": "continue",
+                                "on_action_executed": "fail",
+                            },
+                        },
+                    }
+                ],
+                "second_deployment": True,
+                "success": False,
+                "last_successful_state": "Download",
+            },
+        },
+        {
+            "name": "continue with higher priority",
+            "case": {
+                "maps": [
+                    {
+                        "id": MUID,
+                        "states": {"ArtifactInstall_Enter": {"action": "fail",},},
+                    },
+                    {
+                        "id": MUID2,
+                        "states": {"ArtifactInstall_Enter": {"action": "continue",},},
+                        "priority": 1,
+                    },
+                ],
+                # "continue" has lowest precedence, even if the priority is
+                # higher.
+                "success": False,
+                "last_successful_state": "Download",
+            },
+        },
+        {
+            "name": "force_continue with higher priority",
+            "case": {
+                "maps": [
+                    {
+                        "id": MUID,
+                        "states": {"ArtifactInstall_Enter": {"action": "pause",},},
+                    },
+                    {
+                        "id": MUID2,
+                        "states": {
+                            "ArtifactInstall_Enter": {"action": "force_continue",},
+                        },
+                        "priority": 1,
+                    },
+                ],
+                "success": True,
+                "last_successful_state": "ArtifactCommit",
+            },
+        },
+        {
+            "name": "force_continue with lower priority",
+            "case": {
+                "maps": [
+                    {
+                        "id": MUID,
+                        "states": {"ArtifactCommit_Enter": {"action": "fail",},},
+                    },
+                    {
+                        "id": MUID2,
+                        "states": {
+                            "ArtifactCommit_Enter": {"action": "force_continue",},
+                        },
+                        "priority": -1,
+                    },
+                ],
+                "success": False,
+                "last_successful_state": "ArtifactVerifyReboot",
+            },
+        },
+    ]
+
+    @pytest.mark.min_mender_version("2.7.0")
+    @pytest.mark.parametrize(
+        "case_name,case",
+        [(case["name"], case["case"]) for case in test_update_control_maps_cases],
+    )
+    def test_update_control_maps(
+        self,
+        case_name,
```

Comment:
`case_name` is unused

Reply:
It is required by `parametrize`, otherwise it complains about a missing parameter. The ultimate goal is that the test case be named instead of just an index, which is much harder to identify in the list of cases.